src/components/calendar/helpers.ts

```diff
@@ -81,6 +113,33 @@ export function getDateFormValue(value: Date | null) {
   return value ? value.toISOString() : null;
 }
 
+/**
+ * Converts a DateDateRangeValue object to FormData with
+ * start and end Date values as ISO 8601 strings.
+ * The keys are prefixed with the host name if it exists
+ * and suffixed with 'start' or 'end' accordingly.
+ *
+ * If the date values are null or undefined, the form data values
+ * are empty strings ''.
+ */
+export function getDateRangeFormValue(
```

Comment:
I would move this function in the `form-value.ts` file since it's specific to that behavior.